trl/trainer/ppo_trainer.py

```diff
@@ -171,7 +172,7 @@ def __init__(
             raise ValueError(
                 f"tokenizer must be a PreTrainedTokenizer or PreTrainedTokenizerFast, got {type(tokenizer)}"
             )
-        if not isinstance(model, PreTrainedModelWrapper):
+        if not isinstance(model, (PreTrainedModelWrapper, peft.peft_model.PeftModelForCausalLM)):
```

Comment:
```suggestion
        if not isinstance(model, (PreTrainedModelWrapper, PeftModelForCausalLM)):
```

Reply:
We can also consider using `PeftModel` instead, as it will also contain the seq2seq models